contexts/arbo-context/arbo-filter-update-steps.tsx

```diff
@@ -0,0 +1,29 @@
+import { MapRef } from "react-map-gl"
+import { ArboAction, ArboStateType } from "./arbo-context"
+import { pipe } from "fp-ts/lib/function.js";
+import { adjustMapPositionStep } from "./filter-update-steps/adjust-map-position-step";
+import { updatePediatricAgeGroupFilter } from "./filter-update-steps/update-pediatric-age-group-filter";
+import { addActionToSelectedFiltersStep } from "./filter-update-steps/add-action-to-selected-filters-step";
+import { applyNewSelectedFiltersStep } from "./filter-update-steps/apply-new-selected-filters-step";
+
+export interface HandleArboFilterUpdateInput {
+  state: ArboStateType,
+  action: ArboAction,
+  map: MapRef | undefined
+}
+
+export interface HandleArboFilterUpdateOutput {
```

Comment:
what is the difference between this type and the type above?

Reply:
Yeah you're right, repeating the type like this doesn't really make sense. I'm going to keep two distinct types just in case they are ever different but I'm setting one equal to the other :+1:

ad19f0241cd080a899e982f134caac34b89f93bd